src/lib/moment/compare.js

```diff
@@ -29,11 +29,34 @@ export function isBefore (input, units) {
     }
 }
 
-export function isBetween (from, to, units) {
-    return this.isAfter(from, units) && this.isBefore(to, units);
+export function isBetween(from, to, options) {
+    var units;
+    var inclusive;
+    if (typeof options === 'object') {
+        units = !(isUndefined(options['units'])) ? options['units'] : 'millisecond';
+        inclusive = !(isUndefined(options['inclusive']));
+    } else {
+        units = options;
+    }
+    if (!(isUndefined(inclusive))) {
+        switch (options['inclusive']) {
+            case '()':
+                return this.isAfter(from, units) && this.isBefore(to, units);
+            case '(]':
+                return this.isAfter(from, units) && this.isSameOrBefore(to, units);
+            case '[)':
+                return this.isSameOrAfter(from, units) && this.isBefore(to, units);
+            case '[]':
+                return !(this.isAfter(from, units) && this.isBefore(to, units));
+            default:
+                return 'Check format of inclusion/exclusion.';
```

Comment:
`throw new Error("Inclusive option should be one of '()','[)','(]', or '[]'.");`